### PR TITLE
Delete toolkit scroll bar option

### PR DESCRIPTION
**Pull request by harryfei**
Always use toolkit scrollbar in any GUI config. Simplify the code.

Depends on #919.

### CODE REVIEW

**Comment by harryfei** [comment]
#919 merged.

File: src/frame.c [hunk 1]
```diff
@@ -5592,7 +5592,7 @@ Setting this variable does not affect existing frames, only new ones.  */);
   DEFVAR_LISP ("default-frame-scroll-bars", Vdefault_frame_scroll_bars,
 	       doc: /* Default position of vertical scroll bars on this window-system.  */);
 #ifdef HAVE_WINDOW_SYSTEM
```
[the remark this answers is not in the document]
Comment (harryfei) [reply src/frame.c]: Yeah, you are right. I didn't realize that.

[the remark this answers is not in the document]
Comment (harryfei) [reply src/frame.c]: Fixed

File: src/window.h [hunk 2]
```diff
@@ -804,8 +804,8 @@ window_tool_bar_p(struct window *W);
    || WINDOW_HAS_VERTICAL_SCROLL_BAR_ON_RIGHT (W))
 
 #if (defined (HAVE_WINDOW_SYSTEM)					\
```
[the remark this answers is not in the document]
Comment (harryfei) [reply src/window.h]: Fixed

File: src/frame.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (shaleh) [reply src/frame.c]: A benefit of the simplification you have done.

**Pull request merged**